pangeo-deploy/values.yaml

```diff
@@ -19,18 +19,12 @@ pangeo:
     # See https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/jupyterhub/values.yaml
 
     singleuser:
-      image:
-        name: pangeo/base-notebook
-        tag: 2019.02.01
```

Comment:
These two profiles does not specify an image, which means that you may end up with a default image for zero-to-jupyterhub-k8s's helm chart default values I think.

```python
            {
                'display_name': 'small (n1-highmem-2 | 2 cores, 12GB)',
                'kubespawner_override': {
                    'cpu_limit': 2,
                    'cpu_guarantee': 2,
                    'mem_limit': '12G',
                    'mem_guarantee': '12G',
                }
            },
            {
                'display_name': 'standard (n1-highmem-4 | 4 cores, 24GB)',
                'kubespawner_override': {
                    'cpu_limit': 4,
                    'cpu_guarantee': 4,
                    'mem_limit': '24G',
                    'mem_guarantee': '24G',
                }
            },

Reply:
The default image is set by hubploy. There's some magic here. Basically, all the deployments get an image built by repo2docker at deploy time. In this case, that image is defined here: https://github.com/pangeo-data/pangeo-cloud-federation/tree/staging/deployments/hydro/image